nixos/modules/services/home-automation/mycroft.nix

```diff
@@ -0,0 +1,105 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.mycroft;
+
+  format = pkgs.formats.json {};
+in
+{
+  meta.maintainers = teams.mycroft.members;
+
+  options.services.mycroft = {
+    enable = mkEnableOption "Mycroft, the private and open voice assistant";
+  };
+
+  config = mkIf cfg.enable {
+    users.users.mycroft = {
+      group = "mycroft";
+      home = "/var/lib/mycroft";
```

Comment:
Do you not also need `/opt/mycroft` for skills or does it try to write to different directories as well?

Reply:
If it needs that we should patch the hell out of it.

Reply:
Looks like this is controlled by the `data_dir` option.

https://github.com/MycroftAI/mycroft-core/blob/1ae6900b26e8efb4c014b04cd26572adf10b07b2/mycroft/configuration/mycroft.conf#L93